apps/i18n/studio/en_us.json

```diff
@@ -260,6 +260,42 @@
   "playSoundWinPoint": "play win point sound",
   "playSoundWinPoint2": "play win point 2 sound",
   "playSoundWood": "play wood sound",
+  "playSoundR2D2Sound1": "play R2-D2 1 sound",
```

Comment:
I had it that way first, but it changes the appearance of the block - in that the word `sound` becomes part of the dropdown (where it previously was at the end after the dropdown). So, while this is a little different than droplet (which is redundant with `playSound("R2-D2sound1");)`, I think it is better to be consistent with the blockly block: `play <R2-D2 1> sound`, and that matches well with `play <random> sound` and `play <applause> sound`.